deepchecks/nlp/text_data.py

```diff
@@ -733,6 +739,47 @@ def is_sampled(self, n_samples: t.Optional[int]):
             return False
         return self.n_samples > n_samples
 
+    def describe(self, n_properties_to_show: t.Optional[int] = 4, properties_to_show: t.Optional[t.List[str]] = None):
+        """Provide holistic view of the data.
+
+        Generates the following plots:
+        1. Label distribution
+        2. Statistics about the data such as number of samples, annotation ratio, list of metadata columns, list of
+        text properties and so on.
+        3. Property distribution for the text properties defined either by n_properties_to_show or properties_to_show
+        parameter.
+
+        Parameters
+        ----------
+        n_properties_to_show : int, default 4
+            Number of properties to consider for generating property distribution graphs. If properties_to_show
+            is provided, this value is ignored.
+        properties_to_show : List[str], default None
+            List of property names to consider for generating property distribution graphs. If None, all the
+            properties are considered.
+
+        Returns
+        -------
+        Displays the Plotly Figure.
+        """
+        properties = []
+        if self.properties is not None:
+            if properties_to_show is not None:
+                properties = [prop for prop in properties_to_show if prop in self.properties.columns]
+                if len(properties) != len(properties_to_show):
+                    raise DeepchecksValueError(f'{set(properties_to_show)-set(properties)} propertites does not exist!')
+            else:
+                properties = list(self.properties.columns)[:n_properties_to_show]
+
+        fig = text_data_describe_plot(all_properties_data=self.properties, n_samples=self.n_samples,
+                                      label=self.label, categorical_metadata=self.categorical_metadata,
```

Comment:
I think you should use self.label_for_display here. If I'm right, labels won't be shown correctly for multilabel.
Also, have you checked and made sure this looks good on multilabel and token classification?